pkg/controllers/interface_filter.go

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func FilterMultusNonDefaultInterfaces(ifaces []v1.VirtualMachineInstanceNetworkInterface, networks []v1.Network) []v1.VirtualMachineInstanceNetworkInterface {
+	defaultNetwork := getDefaultNetwork(networks)
+	if defaultNetwork == nil {
+		return ifaces
+	}
+	var secondaryInterfaces []v1.VirtualMachineInstanceNetworkInterface
+	for _, iface := range ifaces {
+		if iface.Name != defaultNetwork.Name {
+			secondaryInterfaces = append(secondaryInterfaces, iface)
+		}
+	}
+	return secondaryInterfaces
+}
+
+func IsDefaultNetwork(net v1.Network) bool {
```

Comment:
should be the last func here please because the follow func uses it

also why it start with Capital ?
we use it only on this file

Reply:
done.